src/Greetings.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Greeting = () => {
+  const [greeting, setGreeting] = useState('');
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchGreeting = async () => {
+      try {
+        const response = await axios.get('http://localhost:3000/messages');
+        setGreeting(response.data.greeting);
+        setError(null);
+      } catch (error) {
+        setError('Error fetching greeting');
+      }
+    };
+
+    fetchGreeting();
+  }, []);
```

Comment:
- [ ] One of the requirements for this project is to connect to the API endpoint using the redux reducer that you created. This means that you should get the random greeting from the store after the reducer has dispatched the `fetchGreeting`. Kindly implement the change. 👍 💻